compose-lint-checks/src/main/java/slack/lint/compose/ViewModelForwardingDetector.kt

```diff
@@ -0,0 +1,81 @@
+// Copyright (C) 2023 Slack Technologies, LLC
+// SPDX-License-Identifier: Apache-2.0
+package slack.lint.compose
+
+import com.android.tools.lint.detector.api.*
```

Comment:
un-wildcard

Reply:
I don't know where this came from, it was disabled in AS 🤦